_posts/2017-11-06-jira-progressbar.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+title: "Projekt-Fortschrittsanzeige in JIRA mittels Scriptrunner"
+date: 2017-11-06 12:00:00
+author: michael
```

Comment:
@jaylinski, did you write the article or Michael?

Reply:
Michael, habs nur für ihn gepushed.

Reply:
passt leicht was nicht :-)